tests/tests_tangd_custom_port.yml

```diff
@@ -0,0 +1,48 @@
+---
+- name: Test tangd_custom_port
+  hosts: all
+  vars:
+    nbde_server_port: 7500
+    nbde_server_firewall_zone: public
+  tasks:
+    - name: install with custom port and firewall zone
+      import_role:
+        name: fedora.linux-system-roles.nbde_server
```

Comment:
I'd think `fedora.` is not needed here as the tests/roles has linux-system-roles.nbde_server/ like this.
```
$ ls tests/roles
caller/  linux-system-roles.nbde_server/
```

Reply:
right - should be `name: linux-system-roles.nbde_server`